.circleci/config.yml

```diff
@@ -3,6 +3,17 @@ version: 2.1
 orbs:
   codecov: codecov/codecov@1.0.5
 
+parameters:
+  db-name:
+    type: "string"
+    default: "testdb"
+  db-user:
+    type: "string"
+    default: "root"
+  db-password:
+    type: "string"
+    default: "Password123"
```

Comment:
Looks like these parameters are duplicated in the new `set_env_variables.sh` script. Could this new script be defined here in the `config.yml` instead to eliminate duplication?